vaadin-grid-flow-parent/vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.js

```diff
@@ -390,6 +390,10 @@ import { isFocusable } from '@vaadin/grid/src/vaadin-grid-active-item-mixin.js';
                     }
                   });
                 });
+
+                // Manually recalculate sorter order indicators in case some of the sorters
+                // were hidden and therefore didn't notify the grid about their direction change.
```

Comment:
```suggestion
                // Manually update sorter order indicators in case some of the sorters
                // were hidden and therefore didn't notify the grid about their direction change.
```